pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/NullHandlingIntegrationTest.java

```diff
@@ -186,4 +186,11 @@ public void testCaseWithIsNotDistinctFrom()
     query = "SELECT description FROM " + getTableName() + " where description IS NOT DISTINCT FROM description";
     testQuery(query);
   }
+
+  @Test
+  public void testTotalCountUsingCountLiteral()
+          throws Exception {
+    String query = "SELECT COUNT(1) FROM " + getTableName();
```

Comment:
I think you need to put `SET enableNullHandling = true;` to verify the desired behavior.